flying-saucer-core/src/main/java/org/xhtmlrenderer/css/parser/property/SizePropertyBuilder.java

```diff
@@ -147,6 +147,35 @@ public List<PropertyDeclaration> buildDeclarations(
             } else {
                 throw new CSSParseException("Invalid value for size property", -1);
             }
+        }  else if (values.size() == 3) {
+            PropertyValue value1 = (PropertyValue) values.get(0);
```

Comment:
Can't we have at least some tests for this code?

It should be easy:

```java
class SizePropertyBuilderTest {
    private final SizePropertyBuilder builder = new SizePropertyBuilder();

    @Test
    void buildDeclarations() {
        PropertyValue cssValue = new PropertyValue(CSS_NUMBER, 2.0f, "foo");
        List<PropertyDeclaration> result = builder.buildDeclarations(COUNTER_INCREMENT, List.of(cssValue), 42, false);
        assertThat(result).hasSize(1);
        assertThat(result).containsExactly(new PropertyDeclaration(COUNTER_INCREMENT, cssValue, false, 42));
    }
}
```

Reply:
I'll try to prepare that, I'm a bit struggling with building the lib in Eclipse...